lib/tasks/sidekiq.rake

```diff
@@ -21,5 +21,15 @@ namespace :sidekiq do
     end
 
   end
+
+  desc 'Clear sidekiq processed and failed queues to save memory'
+  task :clearQueues => :environment do
+
+    require 'sidekiq/api'
+
+    Sidekiq::Stats.new.reset
```

Comment:
Does this do more than reset the counters? I'm not clear on how this would reduce memory usage, or does it actually remove the jobs too?

Reply:
I'm under the impression that this is supposed to clear the jobs from Redis that are completed, based on what I've read but not too sure so experimenting. One of the places was here: https://gist.github.com/wbotelhos/fb865fba2b4f3518c8e533c7487d5354
I need to monitor the memory usage and see if this makes any difference. Just checking if this is an option though or if there's a better approach.

Reply:
Looking at the code - https://github.com/mperham/sidekiq/blob/main/lib/sidekiq/api.rb#L158-L170 - I _think_ it's just changing some keys in Redis which are used to hold the stat information.